### ISSUE TITLE
Dezoomify web doesnt work in android

### ISSUE THREAD

**Issue by agrasena** [lovasoa/dezoomify#524]
Hi, dezoomify's web version can load the image in android chrome and firefox but there is no way to download them except art and culture. Art and culture image after loading give a button with option to download in dezoomfy web browser while other sites don't, so on pc we have to copy image and then paste into ms paint to save but nothing of this sort can be done on android like right click to copy image. Is there way to get button like art and culture images do after fully loading? And there is no app like dezoomify-rs for android too. So is there any way for android user to use dezoomify to download these images. Or if you are working on something for android, maybe an app? Thanks in advance.

**Comment by lovasoa** [comment]
Hello !
  Indeed, there is no way to use dezoomify in android at the moment. Dezoomify cannot create a "download" button if the image tiles do not come from its own servers. Making all tiles go through servers we control (as we do for google arts and culture) would not be free. 

  A solution would be what is described here: https://github.com/lovasoa/dezoomify/issues/337#issuecomment-773498488. 
  Would you be ready to make a small monthly donation to make this work ?

**Comment by Emasoft** [comment]
I have the same problem on Safari (iOS). 
https://gallica.bnf.fr/ark:/12148/bpt6k130641h/f258.item.zoom
There is no way to download the image, even if you can find it as a data blob. Can a bookmarklet solve this with js?

**Comment by lovasoa** [comment]
no, a bookmarklet can't do anything that the site itself can't do

**Comment by Emasoft** [comment]
Can you use javascript to save the composed image on canvas and then on the local storage? 

i.e. [stackoverflow.com](https://stackoverflow.com/questions/20108382/javascript-image-to-base64-for-localstorage)

**Comment by lovasoa** [comment]
A bookmarklet wouldn't add anything that dezoomify can't already do. It doesn't have more permissions. 

And dezoomify doesn't have the permission to access the canvas's pixels  because they come from another origin.
